pulseapi/entries/views.py

```diff
@@ -62,6 +63,28 @@ def toggle_bookmark(request, entryid):
 
 
 @api_view(['PUT'])
+@permission_classes([IsAdminUser])
```

Comment:
this is not quite right, because it assumes only admin users may change this.

However, permissions are handled through policy rules (based on Django "groups"), so rather than using an admin decorator, you will need to use a solution similar to https://github.com/mozilla/network-pulse-api/blob/master/pulseapi/entries/views.py#L65-L100 where the user's permissions are checked to see if they have entry-modification rights.